src/structures/Sticker.js

```diff
@@ -42,16 +46,40 @@ class Sticker extends Base {
     this.name = sticker.name;
 
     /**
-     * The ID of the pack the sticker is from
-     * @type {Snowflake}
+     * The ID of the pack the sticker is from, for standard stickers
+     * @type {?Snowflake}
      */
-    this.packID = sticker.pack_id;
+    this.packID = sticker.pack_id ?? null;
 
     /**
      * An array of tags for the sticker, if any
      * @type {string[]}
      */
     this.tags = sticker.tags?.split(', ') ?? [];
+
+    /**
+     * Whether or not the guild sticker is available
+     * @type {?boolean}
+     */
+    this.available = sticker.available ?? null;
+
+    /**
+     * The ID of the guild that owns this sticker
+     * @type {?Snowflake}
+     */
+    this.guildID = sticker.guild_id ?? null;
+
+    /**
+     * The user that uploaded the guild sticker
+     * @type {?User}
+     */
+    this.user = sticker.user ? this.client.users.add(sticker.user) : null;
```

Comment:
I left this as `user` like the API, but d.js uses `author` for emojis, should it be changed to `author` here too?

Reply:
+1 for me, it's a bit misleading to use `sticker.user`. Although `author` can also be troublesome, because this field is about the user who uploaded it, not the author of the asset itself (see: people who re-upload emojis made by others from other servers). Unless, of course, we ignore that detail.

Depending on what we decide, we'll have to change [`GuildEmoji#author`](https://discord.js.org/#/docs/main/master/class/GuildEmoji?scrollTo=author) (in a later PR) to match.

cc: @iCrawl @SpaceEEC @vladfrangu

Reply:
+1 on `user`

Reply:
+1 on `user`. Also, maybe consider adding a `false` to the `.add` method to not cache the user?

Reply:
> to not cache the user?

🤔 why? aren't users usually cached when received?

also, afaik this is only present when the sticker is fetched in the guild and the bot has perms, so this will currently always be null in d.js (unless the sticker object is manually constructed) since GuildStickerManager hasn't been implemented yet

Reply:
Can you add a .fetch method then?

Reply:
ok looks like nobody responded in #library-discussion so I'll post it here:

![image](https://user-images.githubusercontent.com/11778454/123366203-a648d380-d52c-11eb-9604-240daa4400b3.png)

Reply:
Then... let's do fetchAuthor, ala GuildEmoji? CC @iCrawl @kyranet

Reply:
Sure.